src/quo2/components/community/community_card_view.cljs

```diff
@@ -6,15 +6,11 @@
 
 (defn community-card-view-item
   [{:keys [name description locked
-           status tokens cover tags featured]} on-press]
+           status tokens cover tags width]} on-press]
   [rn/touchable-opacity {:on-press on-press}
-   [rn/view {:style (merge (style/community-card 20)
-                           {:margin-bottom 16}
```

Comment:
cards design should care whether it is `featured` or not.
Also margin should be handled by it's container